python/lsst/afw/cameraGeom/utils.py

```diff
@@ -559,14 +568,14 @@ def showAmp(amp, imageSource=FakeImageDataSource(isTrimmed=False), display=None,
     if overlay:
         with display.Buffering():
             if amp.getHasRawInfo() and ampImSize == amp.getRawBBox().getDimensions():
-                bboxes = [(amp.getRawBBox(), 0.49, afwDisplay.GREEN),]
+                bboxes = [(amp.getRawBBox(), 0.49, afwDisplay.GREEN), ]
```

Comment:
Im not going to insist that you change cases like these, but I think it would be preferable to remove the trailing unneeded comma rather than insert and extra unneeded space after it, just to then close the list. I have noticed this kind of change in a few places.